app-utils/lsof/autobuild/build

```diff
@@ -18,9 +17,6 @@ for file in "${FILES[@]}"; do
     install -Dvm0644 "$SRCDIR/$file" \
         "$PKGDIR"/usr/share/doc/$PKGNAME/$file
 done
-install -Dvm0644 "$SRCDIR"/Lsof.8 \
-    "$PKGDIR"/usr/share/man/man8/lsof.8
 
-abinfo "Installing lsof executable ..."
-install -Dvm0755 "$SRCDIR"/lsof \
-    "$PKGDIR"/usr/bin/lsof 
+abinfo "Installing lsof ..."
+make install DESTDIR="$PKGDIR"
```

Comment:
With this migration, you no longer have to use the `build` script.

Instead, declare `ABTYPE=autotools` in `defines`, and move the out-of-template remainder (documentation installation) to `beyond`. Then, remove the `build` script.